Author: @guimou
Date:   Aug. 16 2021

    Patch Rstudio to use /tmp by default for data-dir and pid file
diff -Nru rstudio-1.4.1717.orig/src/cpp/server/server-options.json rstudio-1.4.1717/src/cpp/server/server-options.json
--- rstudio-1.4.1717.orig/src/cpp/server/server-options.json	2021-08-16 10:33:48.721652283 -0400
+++ rstudio-1.4.1717/src/cpp/server/server-options.json	2021-08-16 10:39:05.585549337 -0400
@@ -54,7 +54,7 @@
             "name": "server-pid-file",
             "memberName": "serverPidFile_",
             "type": "string",
-            "defaultValue": "/var/run/rstudio-server.pid",
+            "defaultValue": "/tmp/rstudio-server.pid",
             "description": "The path to a file where the rserver daemon's pid is written."
          },
          {
@@ -85,7 +85,7 @@
             "name": "server-data-dir",
             "memberName": "serverDataDir_",
             "type": "core::FilePath",
-            "defaultValue": "/var/run/rstudio-server",
+            "defaultValue": "/tmp/rstudio-server",
             "description": "Path to the data directory where RStudio Server will write run-time state."
          },
          {
